### PR TITLE
[elasticsearch]unit test

### PR DESCRIPTION
**Pull request by binbinCheng**
### What is the purpose of the change

Partial unit tests for elasticsearch

### Related issues/PRs

Related issues: #3049
Related pr:#3762

### Brief change log

linkis-engineconn-plugins/elasticsearch

### CODE REVIEW

**Review by jacktao007 — APPROVED** [review]
LGTM.

File: linkis-engineconn-plugins/elasticsearch/src/test/scala/org/apache/linkis/engineplugin/elasticsearch/builder/TestElasticSearchProcessEngineConnLaunchBuilder.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.junit.jupiter.api.{Assertions, Test}
+
+class TestElasticSearchProcessEngineConnLaunchBuilder {
+
+  @Test
+  def testGetExecutorFactories {
+    val engineConnLaunchBuilder = new ElasticSearchProcessEngineConnLaunchBuilder
+    val userCreatorLabel = new UserCreatorLabel
+    val value = engineConnLaunchBuilder.getEngineStartUser(userCreatorLabel)
+    Assertions.assertNotNull(value)
```
Comment (casionone) [inline linkis-engineconn-plugins/elasticsearch/src/test/scala/org/apache/linkis/engineplugin/elasticsearch/builder/TestElasticSearchProcessEngineConnLaunchBuilder.scala:31]: the value is `StorageConfiguration.HDFS_ROOT_USER.getValue`

File: linkis-engineconn-plugins/elasticsearch/src/test/scala/org/apache/linkis/engineplugin/elasticsearch/exception/TestEsConvertResponseException.scala [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+class TestEsConvertResponseException {
+
+  @Test
+  def testEsConvertResponseException: Unit = {
+    val exception = EsConvertResponseException
+    Assertions.assertNotNull(exception)
+  }
+
+}
```
Comment (casionone) [inline linkis-engineconn-plugins/elasticsearch/src/test/scala/org/apache/linkis/engineplugin/elasticsearch/exception/TestEsConvertResponseException.scala:30]: no useful,  remove this test file

File: linkis-engineconn-plugins/elasticsearch/src/test/scala/org/apache/linkis/engineplugin/elasticsearch/exception/TestEsEngineException.scala [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import org.junit.jupiter.api.{Assertions, Test}
+
+class TestEsEngineException {
+
+  @Test
+  def testEsEngineException: Unit = {
+    val exception = EsEngineException
+    Assertions.assertNotNull(exception)
+  }
+
```
Comment (casionone) [inline linkis-engineconn-plugins/elasticsearch/src/test/scala/org/apache/linkis/engineplugin/elasticsearch/exception/TestEsEngineException.scala:29]: no useful,  remove this test file

File: linkis-engineconn-plugins/elasticsearch/src/test/scala/org/apache/linkis/engineplugin/elasticsearch/exception/TestEsParamsIllegalException.scala [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+package org.apache.linkis.engineplugin.elasticsearch.exception
+
+import org.junit.jupiter.api.{Assertions, Test}
+
+class TestEsParamsIllegalException {
+
+  @Test
+  def testEsParamsIllegalException: Unit = {
+    val exception = EsParamsIllegalException
+    Assertions.assertNotNull(exception)
```
Comment (casionone) [inline linkis-engineconn-plugins/elasticsearch/src/test/scala/org/apache/linkis/engineplugin/elasticsearch/exception/TestEsParamsIllegalException.scala:27]: no useful,  remove this test file

File: linkis-engineconn-plugins/elasticsearch/src/test/scala/org/apache/linkis/engineplugin/elasticsearch/factory/TestElasticSearchEngineConnFactory.scala [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+import org.junit.jupiter.api.{Assertions, Test}
+
+class TestElasticSearchEngineConnFactory {
+
+  @Test
+  def testGetExecutorFactories {
+    val elasticSearchEngineConnFactory = new ElasticSearchEngineConnFactory
+    Assertions.assertNotNull(elasticSearchEngineConnFactory.getExecutorFactories)
+  }
+
```
Comment (casionone) [inline linkis-engineconn-plugins/elasticsearch/src/test/scala/org/apache/linkis/engineplugin/elasticsearch/factory/TestElasticSearchEngineConnFactory.scala:29]: no useful,  remove this test file

**Pull request closed without merging**